packages/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:7
+
+# Version of calico-mesos plugin
+ENV CALICO_MESOS_VERSION v0.1.1
+ENV CALICO_DOCKER_VERSION v0.8.0
+
+# Install dependencies
+RUN yum install -y epel-release wget rpm-build
+#RUN yum groupinstall -y "Development Tools"
+#RUN yum install -y tar wget curl docker python-devel python-pip git libffi-devel libssl-devel
```

Comment:
`epel-release` is not needed here - I can remove it. 
`wget` and `rpm-build` are needed to download the neccessary files and call `rpmbuild`, respectively, but are not needed during the build/installation process or needed to run calico-mesos/calicoctl so I do not think it is necessary to add them as dependencies in the spec file.